data/basics/userBalance_test.go

```diff
@@ -147,11 +147,23 @@ func TestEncodedAccountDataSize(t *testing.T) {
 	require.Equal(t, MaxEncodedAccountDataSize, len(encoded))
 }
 
-func TestEncodedAccountAssetsAllocationBound(t *testing.T) {
+func TestEncodedAccountAllocationBounds(t *testing.T) {
```

Comment:
I wasn't thinking that this would be the evolution of my unit test.. but I guess it fits ;-)